start.go

```diff
@@ -27,7 +27,7 @@ your host.`,
 		}
 		switch status {
 		case libcontainer.Created:
-			return container.Signal(libcontainer.InitContinueSignal)
+			return container.Exec()
```

Comment:
I'd rather have the container process remove the FIFO after it unblocks.  Then `start` can always `Exec()`, and you can catch the “FIFO does not exist” error and translate it to a prettier “someone must have already started the container”.